hydromt_wflow/wflow_sediment.py

```diff
@@ -732,15 +729,15 @@ def setup_riverbedsed(
         self,
         bedsed_mapping_fn: str | Path | pd.DataFrame | None = None,
         output_names: Dict = {
-            "river_bottom-and-bank_sediment__median_diameter": "D50_River",
-            "river_bottom-and-bank_clay__mass_fraction": "ClayF_River",
-            "river_bottom-and-bank_silt__mass_fraction": "SiltF_River",
-            "river_bottom-and-bank_sand__mass_fraction": "SandF_River",
-            "river_bottom-and-bank_gravel__mass_fraction": "GravelF_River",
-            "river_water_sediment__kodatie_transport_capacity_a-coefficient": "a_kodatie",  # noqa: E501
-            "river_water_sediment__kodatie_transport_capacity_b-coefficient": "b_kodatie",  # noqa: E501
-            "river_water_sediment__kodatie_transport_capacity_c-coefficient": "c_kodatie",  # noqa: E501
-            "river_water_sediment__kodatie_transport_capacity_d-coefficient": "d_kodatie",  # noqa: E501
+            "river_bottom-and-bank_sediment__d50_diameter": "river_bed_sediment_d50",
```

Comment:
I think merge conflict issue: the wflow variable name changed and should be what is in the update_naming branch